student-work/ashley_riehl/week_1/group_challenge_classes.py

```diff
@@ -0,0 +1,43 @@
+class BaseClass:
+    def __init__(self, first_name, last_name, email):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.email = email
+
+    def ask_a_question(self, question):
+        print(question)
+
+class Student(BaseClass):
+    def __init__(self, first_name, last_name, email, books_dict={}):
+        super().__init__(first_name, last_name, email)
+        self.books_dict = {}
+
+    def add_book(self, book_name, author):
+        self.books_dict[book_name] = author
+
+    def remove_book(self, book_name):
+        del self.books_dict[book_name]
+
+    def show_books(self):
+        print(self.books_dict)
+
+class Teacher(BaseClass):
+    def __init__(self, first_name, last_name, email, student_list=[]):
+        super().__init__(first_name, last_name, email)
+        self.student_list = student_list
+
+    def shuffle_class(self, size):
+        from group_challenge.names_challenge import names_func
```

Comment:
Everything looks good! Only suggestion would be to move the import from inside the shuffle_class method to the top of the file. Usually all imports go up at the top of files with the exception of when you want to use the Python debugger.